src/shared/contexts/csv.worker.ts

```diff
@@ -101,11 +106,29 @@ const fromFlux = (fluxCSV: string): FromFluxResult => {
     let columnDefault: any = ''
 
     for (const chunk of chunks) {
-      tableText = chunk
-        .split('\n')
-        .filter(line => !line.startsWith('#'))
-        .join('\n')
-        .trim()
+      const splittedChunk = chunk.split('\n')
```

Comment:
this looks like the logic from giraffe - is it possible to reuse that?

Reply:
No, because this is the "native" version in the UI. There's a larger problem of having too many `fromFlux` parsers in the UI.

My goal here is simply to make the differences smaller, and not solve the larger problem of removing extraneous `fromFlux` parsers.